domain/transfer/sender_service.go

```diff
@@ -260,6 +258,84 @@ func (s service) CancelNegotiation(ctx context.Context, customerID int, negotiat
 	return negotiation, s.sendNotification(ctx, notification.customer, notification.organizationDID)
 }
 
+func (s service) UpdateTask(ctx context.Context, customer domain.Customer, taskID string, newState string) error {
```

Comment:
Since we only updat the taskStatus, change name to `UpdateTaskState`. In the negotiation flow a task can also by updated with a new `output`